### ISSUE TITLE
esm_master breaks with old info in ~/.esmtoolsrc

### ISSUE THREAD

**Issue by christian-stepanek** [esm-tools/esm_tools#239]
**Request/contribution description**
If a previous version of esm-tools has been used before, in a fresh install of esm-tools v.5 esm_master crashes. This is related to the following information stored in ~/.esmtoolsrc:

```
RUNSCRIPT_PATH=...
NAMELIST_PATH=...
FUNCTION_PATH=...
```
After removing these lines esm_master works as expected.

**Suggested chapter/section in the documents**
I suggest to add a remark to step 5 of "TEN STEPS TO A RUNNING MODEL" to remove respective lines if the an error of the following form occurs:
```
stepanek@ollie0:~/esm-tools_v5/esm_tools$ esm_master 
2020-12-08 17:01:34.682 | DEBUG    | esm_parser.yaml_to_dict:yaml_file_to_dict:93 - IOError (%s) File not found with %s, trying another extension pattern.
2020-12-08 17:01:34.683 | DEBUG    | esm_parser.yaml_to_dict:yaml_file_to_dict:93 - IOError (%s) File not found with %s, trying another extension pattern.
2020-12-08 17:01:34.683 | DEBUG    | esm_parser.yaml_to_dict:yaml_file_to_dict:93 - IOError (%s) File not found with %s, trying another extension pattern.
2020-12-08 17:01:34.683 | DEBUG    | esm_parser.yaml_to_dict:yaml_file_to_dict:93 - IOError (%s) File not found with %s, trying another extension pattern.
2020-12-08 17:01:34.684 | DEBUG    | esm_parser.yaml_to_dict:yaml_file_to_dict:93 - IOError (%s) File not found with %s, trying another extension pattern.
Traceback (most recent call last):
  File "/home/ollie/stepanek/.local/bin/esm_master", line 8, in <module>
    sys.exit(main())
  File "/home/ollie/stepanek/.local/lib/python3.7/site-packages/esm_master/cli.py", line 76, in main
    main_flow(parsed_args, target)
  File "/home/ollie/stepanek/.local/lib/python3.7/site-packages/esm_master/esm_master.py", line 24, in main_flow
    main_infos = GeneralInfos()
  File "/home/ollie/stepanek/.local/lib/python3.7/site-packages/esm_master/general_stuff.py", line 136, in __init__
    self.config = esm_parser.yaml_file_to_dict(CONFIG_YAML)
  File "/home/ollie/stepanek/.local/lib/python3.7/site-packages/esm_parser/yaml_to_dict.py", line 106, in yaml_file_to_dict
    "All file extensions tried and none worked for %s" % filepath
FileNotFoundError: All file extensions tried and none worked for /home/ollie/stepanek/esm-tools_v4/esm_tools/configs/esm_software/esm_master/esm_master.yaml
```
.

**Is this specific to any model/couple setup or to any ESM-Tools functionality?**
No.

**Contribution**
To be added at the end of the paragraph: "In case esm_master crashes with a FileNotFoundError with regard to esm_master.yaml, try to fix this by updating your ~/.esmtoolsrc removing lines that define paths for runscripts, namelists, and functions. Then try again."

**Comment by mandresm** [comment]
Is anyone working on this already? Cause I can just do it very fast.

**Comment by mandresm** [comment]
I close this as the changes are now included in the documentation. Thanks @christian-stepanek for your contribution!

**Comment by pgierz** [comment]
Out of interest, can we get it to **not** break? I had built checks around most of the other parts (at least in the runscripts part) to ensure everything still worked even if you had `FUNCTION_PATH` defined...

**Comment by chrisdane** [comment]
Hi Miguel

I just ran into the same error (having an old `~/.esmtoolsrc`). The note in `ten_steps.rst` about this issue is too far down. I got stuck following the ten steps. 

Also, one solution is simply to remove the already existing `~/.esmtoolsrc` before running `/install.sh`.

I was wondering if this check could be included into the installation process by default? It would clean the ten steps docu.

Cheers,
Chris
